source/roadmap.html.erb

```diff
@@ -17,66 +17,78 @@ title: Roadmap
+          We're completing a procurement exercise to get the best card and pay by bank suppliers. This will help to make sure service teams and paying users have the best value for money and experience when using GOV.UK Pay.
         </p>
-        <h3 class="govuk-heading-m">Accessibility</h3>
-        <p class="govuk-body">We're making sure our service can be used by everyone, following the latest Web Content Accessibility Guidelines 2.2 (WCAG 2.2).</p>
-        <h3 class="govuk-heading-m">Data deletion</h3>
-        <p class="govuk-body">We have reviewed our data retention schedules. We continue to make sure personal data is deleted or redacted within 7 years.</p>
-        <h3 class="govuk-heading-m">Database upgrades</h3>
-        <p class="govuk-body">We are upgrading 5 of our main databases in GOV.UK Pay to the latest Postgres version.</p>
         <h2 class="govuk-heading-l">What's next</h2>
-        <h3 class="govuk-heading-m">Recurring payments improvements</h3>
-        <p class="govuk-body">We'll use feedback from services and paying users to improve how recurring payments work.</p>
-        <h3 class="govuk-heading-m">Improving transaction search</h3>
-        <p class="govuk-body">We're going to make it easier to find and manage transactions in the admin tool.</p>
-        <h3 class="govuk-heading-m">Strengthening our security</h3>
-        <p class="govuk-body">We're getting ready for a new security standard called Payment Card Industry Data Security Standard 4.0 to keep users' information safe.</p>
+        <h3 class="govuk-heading-m">Integrate with new card payment suppliers</h3>
+        <p class="govuk-body">
+          Once we’ve procured new card payment suppliers, we’ll integrate them with GOV.UK Pay.
+        </p>
+        <h3 class="govuk-heading-m">Release pay by bank to a small number of public sector services</h3>
+        <p class="govuk-body">
+          We want to integrate Pay with a pay by bank supplier so we can release pay by bank for a small number of public sector services. This will allow paying users to pay for public sector services with their banking app or online banking. We’ll use what we learn from this work to improve the payment type before releasing it to more public sector services.
+        </p>
+        <h3 class="govuk-heading-m">Complete our 2025 Payment Card Industry assessment</h3>
+        <p class="govuk-body">
+          In June 2025, we’ll complete our 2025 Payment Card Industry Data Security Standard 4.0 assessment to review our processes and make sure we’re keeping users’ information safe.
+        </p>
+        <h3 class="govuk-heading-m">Meet Visa card scheme data field requirements</h3>
+        <p class="govuk-body">
+          Card schemes are doing more to prevent and combat fraud. As a result, from August 2025, Visa require some data fields to be sent with payments. We will need to make changes in Pay to accommodate this for services and tell them about the change.
```

Comment:
```suggestion
          Card schemes are doing more to prevent and combat fraud. As a result, from August 2025, Visa require us to send some data fields with payments. We'll need to make changes in GOV.UK Pay to accommodate this for services and tell them about the change.
```